.github/workflows/lint.yml

```diff
@@ -45,6 +45,7 @@ jobs:
       - name: Run Pylint
         run: |
           python -m tools.pylint \
+            --jobs 1 \
```

Comment:
Prevents duplicate reported results. Fixed in next (currently not released) pylint version